app/js/components/encounters/observations.jsx

```diff
@@ -100,7 +100,7 @@ export default class Observations extends React.Component {
               {
                 filteredObs && filteredObs.length > 0 &&
                 filteredObs.map((observation) => (
-                  <tr className={(observation.voided) ? 'deletedObs' : ''}>
+                  <tr className={(observation.voided) ? 'deletedObs' : 'undeletedobs'}>
```

Comment:
undeletedobs is better named as undeletedObs

Reply:
I don't understand @dkayiwa could you shade more light on this

Reply:
Did you take a look at this? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NamingConventions